projects/1.16/assets/minecolonies/minecolonies/lang/zh_cn.json

```
@@ -1829,13 +1939,132 @@
+  "minecolonies:cook.job.desc": "处理大部分烹饪并且给市民分发食物。",
+  "minecolonies:shepherd.job.desc": "牧羊并控制羊群的规模。",
+  "minecolonies:cowboy.job.desc": "给牛挤奶或喂食来获得牛奶、皮革和生牛肉。",
+  "minecolonies:swineherder.job.desc": "给殖民地提供猪肉。",
+  "minecolonies:chickenherder.job.desc": "收集鸡蛋、羽毛和鸡的各种产物。",
+  "minecolonies:ranger.job.desc": "远程战士，造成高伤害，但是在近距离肉搏中很难存活。",
+  "minecolonies:knight.job.desc": "近距离战士。高抗性，低伤害。",
+  "minecolonies:composter.job.desc": "用一大堆殖民地垃圾堆肥。",
+  "minecolonies:student.job.desc": "读书来增加他们的智力。高智力让他们能更快地学习技能。",
+  "minecolonies:archertraining.job.desc": "正在受训的弓箭手。练习远程战斗。",
+  "minecolonies:combattraining.job.desc": "正在受训的骑士。练习近距离战斗。",
+  "minecolonies:sawmill.job.desc": "为殖民地提供大多数基于木材的物品。",
+  "minecolonies:blacksmith.job.desc": "为殖民地提供大多数工具，护甲和武器。",
+  "minecolonies:stonemason.job.desc": "处理大多数基于石头的配方。",
+  "minecolonies:stonesmeltery.job.desc": "提供各种石头和木炭的冶炼。",
+  "minecolonies:crusher.job.desc": "将圆石粉碎成沙砾，沙砾粉碎成沙子，沙子粉碎成黏土。",
+  "minecolonies:sifter.job.desc": "过滤多种材料来获取好东西。",
+  "minecolonies:florist.job.desc": "为殖民地种植各种各样的花。",
+  "minecolonies:enchanter.job.desc": "通过观察他人来收集魔力。附魔殖民地的工具及制作附魔书。",
+  "minecolonies:researcher.job.desc": "研究玩家选择的主题。",
+  "minecolonies:healer.job.desc": "治疗受伤实体并且为病人治病。",
+  "minecolonies:pupil.job.desc": "坐在周围并学习老师提供的材料。",
+  "minecolonies:teacher.job.desc": "和孩子们坐在一起并提供材料。",
+  "minecolonies:dyer.job.desc": "处理大多数殖民地的染色需求。",
+  "minecolonies:glassblower.job.desc": "提供大多数基于玻璃的物品。",
+  "minecolonies:mechanic.job.desc": "处理红石配方和其他工人不能处理的东西。",
+  "minecolonies:fletcher.job.desc": "处理基于皮革，线和羊毛的配方。",
+  "minecolonies:planter.job.desc": "为殖民地提供甘蔗，仙人掌和竹子。",
+  "minecolonies:rabbitherder.job.desc": "处理一堆兔子。",
+  "minecolonies:concretemixer.job.desc": "制作混凝土粉末和真正的混凝土。",
+  "minecolonies:beekeeper.job.desc": "照料一些蜂巢。",
+  "minecolonies:cookassistant.job.desc": "处理基于食物的、主厨处理不了的配方。",
+  "minecolonies:smelter.job.desc": "将矿石变为粗矿并将粗矿熔炼成原矿。",
+  "minecolonies:druid.job.desc": "支援型单位。给友军提供增益并削弱敌人。",
+  "minecolonies:netherworker.job.desc": "游荡到下界去采集各种材料。是一个非常危险的工作！",
+  "minecolonies:alchemist.job.desc": "酿造药水，种植下界疣和收获槲寄生",
+  "minecolonies:builder.skills.desc": "影响放置和破坏速度。",
+  "minecolonies:deliveryman.skills.desc": "影响行走速度和并发请求执行能力。",
+  "minecolonies:miner.skills.desc": "影响放置和破坏速度。",
+  "minecolonies:quarrier.skills.desc": "影响放置和破坏速度。",
+  "minecolonies:lumberjack.skills.desc": "影响破坏和行走速度。",
+  "minecolonies:farmer.skills.desc": "影响农夫工作速度。",
+  "minecolonies:undertaker.skills.desc": "影响工作速度和复活几率。",
+  "minecolonies:fisherman.skills.desc": "影响钓鱼速度和战利品收获。",
+  "minecolonies:baker.skills.desc": "影响配方改进几率和制造速度。",
+  "minecolonies:cook.skills.desc": "影响烹饪速度。",
+  "minecolonies:shepherd.skills.desc": "影响他们获得的羊毛数量和造成的伤害。",
+  "minecolonies:cowboy.skills.desc": "影响工作速度和造成的伤害。",
+  "minecolonies:swineherder.skills.desc": "影响工作速度和造成的伤害。",
+  "minecolonies:chickenherder.skills.desc": "影响成功获取鸡副产品的几率和击中鸡的几率。",
+  "minecolonies:ranger.skills.desc": "影响速度、准确率和伤害输出。",
+  "minecolonies:knight.skills.desc": "影响生命值、恢复力和攻击速度。",

```

```suggestion
  "minecolonies:knight.skills.desc": "影响生命值、防御力和攻击速度。",
```
resilience并不是原版术语，所以整了个防御力

---

但是恢复力也不算原版术语吧，而且`resilience`原意与防御力还是有点区别的

---

草我一直以为是指的强韧
看实际表现吧